Makefile

```diff
@@ -29,7 +29,7 @@ flake8: check_venv
 	flake8 --max-line-length 120 $(shell find . -name '*.py' -not -path "./venv/*")
 
 install: venv
-	pip install -r requirements.txt
+	( . venv/bin/activate; pip install -r requirements.txt )
```

Comment:
This should fail if activation fails (so `&&` instead of `;`) right?

Reply:
absolutely.